packages/model/addon/-private/model.js

```diff
@@ -1586,17 +1591,18 @@ Model.reopenClass({
    ```app/models/post.js
    import Model, { hasMany } from '@ember-data/model';
 
-   export default Model.extend({
-      comments: hasMany('message')
-    });
+   export default class PostModel extends Model {
+    @hasMany('message') comments;
```

Comment:
looks like these can be indented 2 spaces (or whatever it was before)

Reply:
fixed

Reply:
fixed indent